src/content/docs/guides/startupscripts.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Startup-Scripts
+description: A guide on how to use startup-scripts.
+---
+
+JS-Engine supports startup-scripts, similar to the CSS-snippets from Core-Obsidian.
+
+You can select the script-file you want to use as startup-scripts and JS-Engine will run them for you when never Obsidian starts up.
+
+:::note
+Different from Core-Obsidian, JS-Engine startup-scripts can not be stored inside the `.obsidian/` folder.
+They need to be somewhere inside your vault.
+:::
+
+## Configuring Startup-Scripts
+
+You can configure which files you want to have run from JS-Engines settings tab.
+
+## Global Variables
+
+JS-Engine runs the scripts in the same context as all other code-blocks.
```

Comment:
Small correction, it's not entirely the same. Startup scripts run with a `JSFileExecutionContext` and code blocks run with a `MarkdownCodeBlockExecutionContext`, but that is also not properly documented yet.